src/components/VmActions/index.js

```diff
@@ -142,9 +200,12 @@ VmActions.propTypes = {
 export default connect(
   (state) => ({
     icons: state.icons,
+    VmAction: state.VmAction,
   }),
   (dispatch, { vm }) => ({
+    checkConsole: () => dispatch(checkConsoleInUse({ vmId: vm.get('id') })),
```

Comment:
Please use more self-descriptive name, instead of checkConsole use i.e. onCheckConsoleSessionInUse